indexer/accounttx/accounttx.go

```diff
@@ -0,0 +1,209 @@
+package accounttx
+
+import (
+	"encoding/json"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ignite/cli/ignite/pkg/cosmoscmd"
+	abci "github.com/tendermint/tendermint/abci/types"
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	tm "github.com/tendermint/tendermint/types"
+	tmdb "github.com/tendermint/tm-db"
+	terra "github.com/terra-money/alliance/app"
+	"github.com/terra-money/mantlemint/db/safe_batch"
+	"github.com/terra-money/mantlemint/indexer"
+	"github.com/terra-money/mantlemint/indexer/tx"
+	"github.com/terra-money/mantlemint/mantlemint"
+)
+
+var cdc = cosmoscmd.MakeEncodingConfig(terra.ModuleBasics)
```

Comment:
This does not appear scalabable to any chain...
I think this should come from App.AppCodec